packages/assistant-stream/src/core/modules/assistant-stream.ts

```
@@ -0,0 +1,230 @@
+    let [stream, controller] = createToolCallStreamController();
+    this._addPart({ type: "tool-call", toolName, toolCallId }, stream);
+
+    if (opt.args !== undefined) {
+      controller.argsText.append(JSON.stringify(opt.args));
+      controller.argsText.close();
+    }
+    if (opt.result !== undefined) {
+      controller.setResult(opt.result);
+    }
+
+    return controller!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)